contracts/minter/src/contract.rs

```diff
@@ -432,14 +430,14 @@ pub fn execute_update_start_time(
         return Err(ContractError::AlreadyStarted {});
     }
 
-    let genesis_start_time = Expiration::AtTime(Timestamp::from_nanos(GENESIS_MINT_START_TIME));
+    let genesis_start_time = Timestamp::from_nanos(GENESIS_MINT_START_TIME);
     let start_time = if start_time < genesis_start_time {
         genesis_start_time
     } else {
         start_time
```

Comment:
we are not checking that the new `start_time > env.block` time right ?

Reply:
also maybe for updating doesn't makes sense to default to genesis time, just in instantiation. You have to be explicit about the time you want to update to.

Reply:
Also fixed this.